src/pages/en/guides/deploy/github.md

```diff
@@ -44,6 +48,7 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
               node-version: 16
 
           # Not using npm? Change `npm ci` to `yarn install` or `pnpm i`
+          # to run npm ci you must have a package-lock.json file in your repository.
```

Comment:
I bet lots of people won’t read the workflow file in detail, so maybe we should move this to a `:::caution` somewhere?

```md
:::caution
This workflow uses the `npm ci` command by default. You must include a `package-lock.json` file in your repository for this to work. To generate one, run `npm i` in your terminal and commit the resulting lock file.
:::
```

Reply:
Yeah! The other option might be another step in the instructions that says something like:

```md
1. Make sure your repository includes `package-lock.json`. Don’t see it? Run `npm i` to generate one and then commit it.
```

But that breaks the explicit link with `npm ci` a bit, so I’d probably go with the caution. It’s probably going to look terrible because of how we space things, but that’s something on my to-do list to improve anyway!

Reply:
I moved to the end of the code block